terminology/src/main/java/au/csiro/pathling/terminology/DefaultTerminologyServiceFactory.java

```diff
@@ -121,25 +121,25 @@ private DefaultTerminologyService2 createService2() {
   }
 
   private static CloseableHttpClient buildHttpClient(
-      @Nonnull final HttpClientConf clientConf,
-      @Nonnull final HttpCacheConf cacheConf) {
+      @Nonnull final HttpClientConfiguration clientConfig,
+      @Nonnull final HttpClientCachingConfiguration cacheConfig) {
     final PoolingHttpClientConnectionManager connectionManager = new PoolingHttpClientConnectionManager();
-    connectionManager.setMaxTotal(clientConf.getMaxConnectionsTotal());
-    connectionManager.setDefaultMaxPerRoute(clientConf.getMaxConnectionsPerRoute());
+    connectionManager.setMaxTotal(clientConfig.getMaxConnectionsTotal());
+    connectionManager.setDefaultMaxPerRoute(clientConfig.getMaxConnectionsPerRoute());
     final HttpClientBuilder clientBuilder;
-    if (cacheConf.isEnabled()) {
-      final CacheConfig cacheConfig = CacheConfig.custom()
-          .setMaxCacheEntries(cacheConf.getMaxCacheEntries())
-          .setMaxObjectSize(cacheConf.getMaxObjectSize())
+    if (cacheConfig.isEnabled()) {
+      final CacheConfig httpClientCacheConfig = CacheConfig.custom()
+          .setMaxCacheEntries(cacheConfig.getMaxEntries())
+          .setMaxObjectSize(cacheConfig.getMaxObjectSize())
           .build();
-      clientBuilder = CachingFactories.of(cacheConf.getStorageType())
-          .create(cacheConfig, cacheConf.getStorage());
+      clientBuilder = CachingFactories.of(cacheConfig)
+          .create(httpClientCacheConfig);
     } else {
```

Comment:
`cacheConfig` and  `httpClientCacheConfig` are a little bit confusing here. 
Maybe 'cachingConfiguration` and `cacheConfig` instead.

Reply:
Fixed in a3489de5588724fdcc8515543b969dffb7d9cf5f.